src/js/claims-status/containers/AdditionalEvidencePage.jsx

```diff
@@ -64,23 +64,33 @@ class AdditionalEvidencePage extends React.Component {
     this.props.router.push(`your-claims/${this.props.claim.id}/files`);
   }
   render() {
+    const breadcrumbArr = [
+      { key: 'home', href: '/', text: 'Home' },
+      { key: 'disability-benefits', href: '/disability-benefits/', text: 'Disability Benefits' },
+      { key: 'your-claims', href: '/track-claims/your-claims/', text: 'Track Your Claims and Appeals' }
+    ];
+
+    const listItem = breadcrumbArr.map(item => {
+      return <li key={item.key}><a href={item.href}>{item.text}</a></li>;
+    });
```

Comment:
This is kinda picky, but `listItem` seems vague to me. Maybe just `breadcrumbItems` or something?